drakrun/drakrun/scripts/cfg.template

```diff
@@ -1,11 +1,16 @@
+# DomU config template for DRAKVUF Sandbox
+
+# This is the template used to generate DomU configuration instances.
+# Please don't edit this file unless you know what you are doing.
+
 arch = 'x86_64'
 name = "vm-{{ VM_ID }}"
-maxmem = 3048
-memory = 3048
-vcpus = 2
-maxvcpus = 2
+maxmem = {{ MEMORY }}
+memory = {{ MEMORY }}
+vcpus = {{ VCPUS }}
+maxvcpus = {{ VCPUS }}
 type = "hvm"
-boot = "cd,menu=on,splash=/usr/share/drakrun/splash.jpg,splash-time=2000"
+boot = "cd"
```

Comment:
If we're removing the splash screen, we should also delete it from repository and deb scripts

Reply:
Unfortunately it never really worked due to some complicated Xen/qemu related issues :F As this is a deadcode, I want to remove it.

:+1: I will also remove from the repository and deb scripts.